tests/sar_pipeline/isce3_rtc/test_full_docker_build_and_run.py

```diff
@@ -294,3 +485,127 @@ def test_docker_dual_pol_with_args():
     assert (
         result.returncode == 0
     ), f"Non-zero exit code: {result.returncode}\nSTDOUT:\n{result.stdout}\nSTDERR:\n{result.stderr}"
+
+
+# folders for products to compare
+TEST_2_RTC_S1_STATIC_S3_PROJECT_FOLDER = (
+    f"{TEST_S3_PROJECT_FOLDER}/{TEST_2_S3_RTC_S1_STATIC_PRODUCT_SUBPATH}"
+)
+TEST_2_COMPARE_RTC_S1_STATIC_S3_PROJECT_FOLDER = (
+    f"{PERSISTENT_S3_PROJECT_FOLDER}/{TEST_2_S3_RTC_S1_STATIC_PRODUCT_SUBPATH}"
+)
+TEST_2_RTC_S1_STATIC_LOCAL_COMPARISON_OUTPUTS_DIR = (
+    f"{LOCAL_COMPARISON_OUTPUTS_DIR}/TEST_2_RTC_S1_STATIC"
+)
+
+
+def test_compare_dual_pol_rtc_s1_static_product_outputs():
+    """
+    This function will compare the outputs created in the test_docker_single_pol_with_args
```

Comment:
Specify that this is for static layers, and update the function name in the docstring to say "dual_pol"